artifactory/commands/utils/testdata/transferconfig/virtual_repo_a.json

```diff
@@ -0,0 +1,48 @@
+{
+  "key": "a-virtual",
+  "packageType": "maven",
+  "description": "virtual repository with members",
+  "includesPattern": "**/*",
+  "excludesPattern": "",
+  "repoLayoutRef": "maven-2-default",
+  "signedUrlTtl": 90,
+  "enableComposerSupport": false,
+  "enableNuGetSupport": false,
+  "enableGemsSupport": false,
+  "enableNpmSupport": false,
+  "enableBowerSupport": false,
+  "enableChefSupport": false,
+  "enableCocoaPodsSupport": false,
+  "enableConanSupport": false,
+  "enableDebianSupport": false,
+  "debianTrivialLayout": false,
+  "ddebSupported": false,
+  "enablePypiSupport": false,
+  "enablePuppetSupport": false,
+  "enableDockerSupport": false,
+  "dockerApiVersion": "V2",
+  "blockPushingSchema1": true,
+  "forceNugetAuthentication": false,
+  "forceP2Authentication": false,
+  "forceConanAuthentication": false,
+  "enableVagrantSupport": false,
+  "enableGitLfsSupport": false,
+  "enableDistRepoSupport": false,
+  "priorityResolution": false,
+  "projectKey": "default",
+  "environments": [],
+  "repositories": ["b-local"],
```

Comment:
Shouldn't that be `b-virtual`? :)
I suggest putting this field on top so that the difference between the configs is clear